setup-environment-internal

```diff
@@ -262,13 +262,29 @@ ln -sf "${MANIFESTS}" "${OEROOT}"/layers/
 
 DISTRO_DIRNAME=$(echo "${DISTRO}" | sed 's#[.-]#_#g')
 
-if [ -z "$LMP_VERSION_CACHE" ]; then
```

Comment:
We need to keep this check because if the variable are specified the version contained on it  will be the one we will use.
Maybe we need to discover the LMP_PARTNER_NAME before this check so that we can also use it if necessary.

Reply:
you mean we should keep the name `LMP_VERSION_CACHE`?

Reply:
yes, we should keep it. 
this is an external variable and if provide we use the version on it and nothing more.
but when on a partner layer we need to also discover the partner name.

Reply:
fixed